src/app/lib/utilities/formatDuration.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable prefer-template */
+
+const lpad = s => s.toString().padStart(2, '0');
```

Comment:
Our babel doesn't polyfil `padStart` :O

Reply:
We checked again and it *was* polyfilling `padStart`.

False alarm